### ISSUE TITLE
adapt-contrib-mcq reads checkbox checked when at maximum selections on a new selection

### ISSUE THREAD

**Issue by chris-steele** [adaptlearning/adapt_framework#1837]
### Issue
This is only an issue when `1 < _selectable  < _items.length`. Noticed when using screen reader as the input elements become checked (but not visually).

### To reproduce
Select maximum options in a multiple choice mcq, then select one more. Listen to the screen reader.

**Comment by chris-steele** [comment]
I've tried setting unselected `input` items to `disabled` when the user is not allowed to select any further items, but the potential issue with this approach is that a screen reader user then no longer hears the disabled items.

@oliverfoster, @kirsty-hames, @ryan-lynch what are your thoughts on this?

**Comment by ryan-lynch** [comment]
Hi @chris-steele can you clarify the scenario in which `1 < _selectable < _items.length` please?

**Comment by chris-steele** [comment]
@ryan-lynch we've had some scenarios from time to time where a question has, say, 4 items, but the author requires that the user select only 1 or 2, so `_selectable` is set to 2. I'm trying to recall an example to put this in context, but cannot at the moment

**Comment by oliverfoster** [comment]
i'm not sure we can provide a solution to this. the instruction text should provide explicit instruction to the user.

**Comment by oliverfoster** [comment]
ok... so update on this one:

the screen reader reads the checkbox as "checkbox checked" event though the checkbox doesn't get ticked because it's over the selectable limit.

**Comment by ryan-lynch** [comment]
That's an interesting one @oliverfoster - I suppose we may need to set the checkbox to aria hidden when the selectable limit has been reached. Or potentially change dynamically update it's aria-label label so that it reads something relevant like "selectable limit reached".

Do you guys have any thoughts?

**Comment by oliverfoster** [comment]
was thinking a preventDefault on the click. i'm going to mess around with it, see if i can get it behave. aria-hidden and disabled will make the checkbox stop reading, which we don't want. we just want it not to say "checkbox checked".... i think?
